diff --git a/tools/fetch_nodejs.mts b/tools/fetch_nodejs.mts
index cee7f2c4..c96c5911 100644
--- a/tools/fetch_nodejs.mts
+++ b/tools/fetch_nodejs.mts
@@ -36,6 +36,7 @@ async function fetchSingleVersion(platformInfo, nodeVersion) {
 		isZip ? 'zip' : 'tar.gz'
 	}`
 	const tarPath = path.join(cacheDir, tarFilename)
+        console.log(tarPath)
 	if (!(await fs.pathExists(tarPath))) {
 		const tarUrl = `https://nodejs.org/download/release/v${nodeVersion}/${tarFilename}`
 
